tests/test_plugin_loader.py

```
@@ -126,6 +129,64 @@ def tearDown(self):
+        sys.modules["schedule"] = original_schedule
+    else:
+        sys.modules.pop("schedule", None)
+    importlib.reload(pl_module)
+
+
+def test_temp_sys_path_handles_missing_remove():
+    """_temp_sys_path should swallow ValueError when path removal fails."""
+    original_path = sys.path
+
+    class PathList(list):
+        def remove(self, value):  # noqa: A003 - intentional shadow for list.remove
```

Ruff (0.14.10)

Unused method argument: `value`

(ARG002)